xaitk_saliency_demo/app/engine/ml_models.py

```diff
@@ -59,16 +53,22 @@ def update_ml_device(cpu_only=True):
     "SimilarityAlexNet",
     "SimilarityVgg16",
     # detection ------------
-    "DetectionFasterRCNN",
-    "DetectionRetinaNet",
+    "DetectionFRCNN",
+    "DetectionCenterNetVisdrone",
 ]
 
 # -----------------------------------------------------------------------------
-# Perform monkey patch to get class probabilities
+# Model file acquisition / caching
 # -----------------------------------------------------------------------------
 
-RoIHeads.postprocess_detections = faster_rcnn_postprocess_detections
-RetinaNet.postprocess_detections = retinanet_postprocess_detections
+CENTERNET_RESNET50 = grabdata(
+    "https://data.kitware.com/api/v1/item/623259f64acac99f426f21db/download",
+    fname="centernet-resnet50.pth",
+    appname="xaitk-saliency-demo",
+    hash_prefix="a0083ec55d46c420d06c414e5ecc1863d6ad9b6a1732acff5c9dba28158a4"
+                "c5a04f43541415d503fa776031a7329e3912864ae2348b3bee035df0d1e7a"
+                "cefa49"
+)
```

Comment:
These are VisDrone weights. I am not aware of any COCO weights, though this would be at odds with the current implementation this is for (embeds visdrone labels, maybe other things).